source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -1248,6 +1248,63 @@ CFDDEMSolver<dim>::postprocess_fd(bool first_iteration)
     }
 }
 
+template <int dim>
+void
+CFDDEMSolver<dim>::postprocess_cfd_dem()
+{
+  // Calculate total volume of fluid and solid 
+  if (this->simulation_parameters.post_processing.calculate_volume_phases)
+   {
+      TimerOutput::Scope t(this->computing_timer, "total_volume_calculation");
+      double             total_volume_fluid, total_volume_particles;
+      std::tie(total_volume_fluid, total_volume_particles) = calculate_fluid_and_particle_volumes(
```

Comment:
Here I used the name "calculate_fluid_and_particle_volume", which is very specific, but I think this name is too long for the parameter. That's why I applied the name "total_volume_phases" to the parameter, the name of output file, and other arguments.
But the name "total_volume_phases" can also be applied to VOF, so I feel like I need to come up with something good and short